src/Login.js

```diff
@@ -43,7 +43,7 @@ class Login extends Component {
 
     return (
       <div className={pageStyles}>
-        <p style={{ color: "red" }}>{this.state.loginError ? 'Email or password incorrect.' : ''}</p>
+        <p style={{ color: "red", height: "20px" }}>{this.state.loginError ? 'Email or password incorrect.' : ''}</p>
```

Comment:
I know it was me being lazy originally with this inline styling, but now that there are a couple styles adding up for this element, could we move these CSS rules to react-emotion like the rest of the styles?